File: 2° Semestre/Lista Java/meses.java
import java.util.Scanner;
class Main 
{
  public static void main(String[] args) 
  {
    Scanner ler= new Scanner (System.in);
    int mes;
    System.out.printf("digite um número inteiro de 1 a 12:");
    mes= ler.nextInt();

    switch (mes)
    {
      case 1: System.out.printf("JANEIRO");break;
      case 2: System.out.printf("FEVEREIRO");break;
      case 3: System.out.printf("MARÇO");break;
      case 4: System.out.printf("ABRIL");break;
      case 5: System.out.printf("MAIO");break;
      case 6: System.out.printf("JUNHO");break;
      case 7: System.out.printf("JULHO");break;
      case 8: System.out.printf("AGOSTO");break;
      case 9: System.out.printf("SETEMBRO");break;
      case 10: System.out.printf("OUTUBRO");break;
      case 11: System.out.printf("NOVEMBRO");break;
      case 12: System.out.printf("DEZEMBRO");break;
      default: System.out.printf("Mês inválido");break;
    }
  }
}

File: 2° Semestre/Lista C/meses.c
#include <stdio.h>
int main(void) 
{
  int mes;
  printf("digite um número inteiro de 1 a 12:");
  scanf("%d",&mes);

  if(mes == 1) printf("JANEIRO");
  else if (mes==2) printf("FEVEREIRO");
       else if (mes==3) printf("MARÇO");
            else if (mes==4) printf("ABRIL");
                 else if (mes==5) printf("MAIO");
                      else if (mes==6) printf("JUNHO");
                           else if (mes==7) printf("JULHO");
                                else if (mes==8) printf("AGOSTO");
                                     else if (mes==9) printf("SETEMBRO");
                                          else if (mes==10) printf("OUTUBRO");
                                               else if (mes==11) printf("NOVEMBRO");
                                                    else if (mes==12) printf("DEZEMBRO");
                                                         else printf("Número inválido");
  return 0;
}